src/components/cart/item/total-cost.ts

```diff
@@ -0,0 +1,10 @@
+import { formatPrice } from '../../../helpers/format-price';
+import { CustomEventEmitter } from '../../../utils/event-emitter';
+
+export const costEventEmitter = new CustomEventEmitter();
+
+export function createTotalCostEmitter(span: HTMLSpanElement): void {
+  costEventEmitter.subscribe('total-cost', (...arguments_): void => {
+    span.textContent = arguments_[0] === 0 ? '' : `Total to be paid: ${formatPrice(Number(arguments_[0]))}`;
+  });
```

Comment:
FYI: You can do it like this and it stops throwing an error
```suggestion
export function createTotalCostEmitter(span: HTMLSpanElement): void {
  costEventEmitter.subscribe('total-cost', (sum) => {
    span.textContent = sum === 0 ? '' : `Total to be paid: ${formatPrice(Number(sum))}`;
  });
}
```

Reply:
nice. done